WorkflowTesting/Sources/Internal/RenderTester+TestContext.swift

```diff
@@ -114,4 +120,26 @@
         }
     }
 
+    func defaultRendering<Child>(for workflowType: Child.Type) -> Child.Rendering? where Child: Workflow {
```

Comment:
Are these used?

Reply:
Oh whoops, these were my failed attempts at avoiding the `() as! Child.Rendering`. Removing.